concrete/src/User/UserList.php

```diff
@@ -156,6 +156,20 @@ public function filterByIsActive($isActive)
         $this->query->setParameter('uIsActive', $isActive);
     }
 
+    /**
+    * Filter list by whether a user is validated or not
+    * @param $val
```

Comment:
We could indicate here the variable name (`$isValidated`) and its type (`boolean`).

So this would look as: `@param boolean $isValidated`